.changeset/sharp-flowers-relax.md

```diff
@@ -0,0 +1,15 @@
+---
+"@salt-ds/lab": minor
+---
+
+Add `SegmentedButton` to labs
+
+```tsx
+const SegmentedButton = () => (
```

Comment:
"SegmentedButton" isn't a good name for what this component is doing. Personally I would rename it to SegmentedButtonGroup or Segmented Group. You should chat with @yunjungyeh

Reply:
To me A SegmentedButton suggests a button that is divided into multiple sections so i think it makes sense. Does a SegmentedButtonGroup not suggest a group of buttons all of which are all divided.
@yunjungyeh What do you think ?

Reply:
I would rather it be called a group. Even If we simplify it and call it a ButtonGroup